src/FlipMove.js

```diff
@@ -180,15 +194,15 @@ class FlipMove extends Component {
     // into the nextChildren in its ORIGINAL position. Note that, as we keep
     // inserting old items into the new list, the "original" position will
     // keep incrementing.
-    let numOfChildrenLeaving = 0;
-    this.state.children.forEach((child, index) => {
-      const isLeaving = !nextChildren.find(({ key }) => key === child.key);
+    let numOfChildrenLeaving: number = 0;
```

Comment:
This seems like an annotation that Flow could infer, unless there's a reason that you wanted to be explicit here?